ocs_ci/ocs/cluster.py

```diff
@@ -2788,3 +2822,29 @@ def get_lvm_full_version():
     image = getattr(redhat_operators_catalogesource_ocs, "data")["spec"]["image"]
     full_version = image.split(":")[1]
     return full_version
+
+
+def get_lvm_pods_names():
```

Comment:
1. function returns pods object, not pods name, so i suggest to change function name accordingly
2. such functions can be found [here](https://github.com/red-hat-storage/ocs-ci/blob/5192d7601f9e8bc709c70b3dedba09afa97d31c3/ocs_ci/ocs/resources/pod.py#L693), so consider move it there

Reply:
You are right. Will change the function name and move it. Thanks!